test/bash_test_wrapper.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# CATEGORY is compilable, fail_compilation or runnable
+category=$1
+test_name=$2
+script_name=${category}/${test_name}.sh
+
+echo " ... /${script_name}"
+
+output_file=${RESULTS_DIR}/${category}/${test_name}.out
+
+./${script_name} > ${output_file}  2>&1
+if [ $? -ne 0 ]; then
+    # duplicate d_do_test output
+    echo >> ${output_file}
+    echo ============================== >> ${output_file}
+    echo Test failed >> ${output_file}
```

Comment:
It's good to state here which test failed as the tests might run in high parallelism.